{{cookiecutter.project_name}}/CITATION.cff

```diff
@@ -1,7 +1,7 @@
 # YAML 1.2
 ---
 cff-version: "1.1.0"
-title: "{{ cookiecutter.project_name }}"
+title: "{{ cookiecutter.directory_name }}"
```

Comment:
```suggestion
title: "{{ cookiecutter.package_name }}"
```

Reply:
Dont like the suggestion for example when you have
- directory_name = pymatchms
- package_name = matchms

You will publish packages as pymatchms on zenodo/pypi/sonarcloud/readthedocs

Reply:
We chose not to include these suggestions in this PR